test/optim/test_optim.py

```diff
@@ -1286,6 +1287,20 @@ def test_adamw_complex(self):
             self._test_complex_2d(functools.partial(AdamW, foreach=foreach, weight_decay=0.2))
             self._test_complex_2d(functools.partial(AdamW, foreach=foreach, weight_decay=0.2, amsgrad=True))
 
+    def test_adamw_serialization(self):
+        model = torch.nn.Linear(5, 5)
+        optim = torch.optim.AdamW(model.parameters())
+
+        buffer = io.BytesIO()
+        torch.save(optim.state_dict(), buffer)
+        buffer.seek(0)
+        loaded_dict = torch.load(buffer)
+
+        new_optim = torch.optim.AdamW(model.parameters())
```

Comment:
hmmm do you think the test could be more interesting if this were torch.optim.Adam instead? The flag should still be on